src/main/java/org/jboss/tools/intellij/openshift/ui/component/CreateComponentDialogStep.java

```diff
@@ -274,7 +281,7 @@ private void loadModel() {
             for (TreeNode nodeObject : Collections.list(rootNode.children())) {
                 DefaultMutableTreeNode node = (DefaultMutableTreeNode) nodeObject;
                 ComponentType type = (ComponentType) node.getUserObject();
-                if (type.getName().equals(model.getComponentTypeName())) {
+                if (type.getName().equals(model.getSelectedComponentType().getName())) {
```

Comment:
I ran into an issue where there is multiple registry with same name, and when I want to switch from the default selected one to another. the UI still select the first one.